### PR TITLE
Hulk nerf port

### PR DESCRIPTION
**Pull request by MacHac**
### Intent of Pull Request
- Hulk punches no longer stun players.

Ported hulk nerf from tgstation at the request of Adam El-Tablawy.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
This kills the geneticist

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	if(user.a_intent == "harm")
 		..(user, 1)
 		adjustBruteLoss(5)
-		Weaken(4)
+		//Weaken(4)
```
Comment (Super3222) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:5]: /TG/ was able to balance removing stunning by making the hulk do 15 brute damage per punch. By taking out the weaken and simply allowing it do 5 brute, basically makes hulks do normal punches.

**Comment by ghost** [comment]
Yea, give hulk 15 damage per punch, not 5.

Suitable nerf.

**Comment by MacHac** [comment]
Buffed hulk damage following feedback.

**Comment by FluffySurvivor** [comment]
Make punches deal stamina damage as well. Hulks cannot use ranged CC.

**Comment by DavidEdwards** [comment]
For now this is fine, lets see how hulks go with this change. If it is too far, we can reinvestigate this issue.

**Pull request merged**